code/game/gamemodes/traitor/traitor.dm

```diff
@@ -117,6 +130,27 @@
 			else
 				special_role_text = "antagonist"
 
+			var/datum/antagonist/traitor/contractor/contractor = traitor.has_antag_datum(/datum/antagonist/traitor/contractor)
+			if(contractor?.contractor_uplink)
+				var/count = 1
+				var/earned_tc = contractor.contractor_uplink.hub.reward_tc_paid_out
+				for(var/c in contractor.contractor_uplink.hub.contracts)
+					var/datum/syndicate_contract/C = c
+					// Locations
+					var/locations = list()
+					for(var/a in C.contract.candidate_zones)
+						var/area/A = a
+						locations += (A == C.contract.extraction_zone ? "<b><u>[A.map_name]</u></b>" : A.map_name)
+					locations = jointext(locations, ", ")
```

Comment:
```suggestion
					locations = english_list(locations)
```
Try this to see if it gets a cleaner output.

Also please assign the text to its own text value instead of re-assigning the list.

Reply:
Before: `a, b, c`
After: `a, b and c`

Much better